.github/workflows/code-quality.yml

```diff
@@ -33,5 +33,8 @@ jobs:
         with:
           version: v1.54.2
 
+      - name: Install Go CI tools
+        run: cat tools.go | grep _ | awk -F'"' '{print $$2}' | xargs -tI % go install %
+
```

Comment:
This is not necessary. Please read https://www.jvt.me/posts/2022/06/15/go-tools-dependency-management/